persistence/src/main/java/de/starwit/persistence/entity/RelationshipType.java

```diff
@@ -0,0 +1,40 @@
+package de.starwit.persistence.entity;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum RelationshipType {
+    OneToOne("one-to-one"),
+    OneToMany("one-to-many"),
+    ManyToOne("many-to-one"),
+    ManyToMany("many-to-many");
+
+    private String jsonFormat;
+
+    private static final Map<String, RelationshipType>  lookup = new HashMap<>();
+
+    static
```

Comment:
Could you please explain, what the following code block does? I did not see anything like this before. Thanks!